deploy/crds/saas.3scale.net_zyncs_crd.yaml

```diff
@@ -0,0 +1,217 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: zyncs.saas.3scale.net
+spec:
+  group: saas.3scale.net
+  names:
+    kind: Zync
+    listKind: ZyncList
+    plural: zyncs
+    singular: zync
+  scope: Namespaced
+  subresources:
+    status: {}
+  version: v1alpha1
```

Comment:
Can we put this block:
```yaml
  version: v1alpha1
  versions:
  - name: v1alpha1
    served: true
    storage: true
```
 At the end of the file, like all the other SaaS CRDs (actually by default, `operator-sdk` put it at the end)? so when we need to update it to `v1`, it will be easier to do so if all of them look like the same

Reply:
Moved to the end in https://github.com/3scale/saas-operator/pull/5/commits/4b04fffd158628bcbd5207d74bcdf7b494c08e13